### PR TITLE
Add documentation for the --reg-winsvc-auto-start and --reg-winsvc-delay-start flags.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Follow-up to fluent/fluentd#3037.

### CODE REVIEW

File: install/install-by-msi.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ server process. Please execute `Td-agent Command Prompt` again but with
 administrative privilege, and type the two commands below.
 
 ```
-> fluentd --reg-winsvc i
+> fluentd --reg-winsvc i --reg-winsvc-auto-start --reg-winsvc-delay-start
 > fluentd --reg-winsvc-fluentdopt '-c C:/opt/td-agent/etc/td-agent/td-agent.conf -o C:/opt/td-agent/td-agent.log'
 ```
```
Comment (cosmo0920) [inline install/install-by-msi.md:80]: I think that example should be minimal.
`--reg-winsvc-auto-start` and `--reg-winsvc-delay-start` should be in **NOTE:**.

---

**NOTE:** Automatic td-agent service requires the additional commandline parameters:

```
>  fluentd --reg-winsvc i --reg-winsvc-auto-start --reg-winsvc-delay-start
```

And then, register fluentd opt with `--reg-winsvc-fluentdopt` option like as the above example.

Comment (igorpeshansky) [reply install/install-by-msi.md:80]: Done. Is this what you had in mind?

**Comment by igorpeshansky** [comment]
Thanks. Sure, I can rebase. Should I squash the commits while I'm at it, or leave them as two separate ones?

**Comment by cosmo0920** [comment]
Well don't recommend to squash commits. Both of T.two commits or squashing the one commits are acceptable.

**Comment by igorpeshansky** [comment]
Thanks, I've just rebased off the 1.0 branch. Had to add a separate commit to prevent merge conflicts. Please take another look.

**Comment by igorpeshansky** [comment]
Great, thanks. Is this ready to merge?

**Pull request merged**